src/test/java/org/folio/calendar/integration/api/health/BaseApiInitializationTest.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.calendar.integration.api.health;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.folio.calendar.integration.BaseApiTest;
+import org.junit.jupiter.api.Test;
+
+class BaseApiInitializationTest extends BaseApiTest {
+
+  @Test
+  void testContextLoads() {
+    assertThat(metadata)
+      .as("Ensure application context starts and database can be initialized")
+      .isNotNull();
```

Comment:
This test seems a little odd to me, wouldn't this failure happen in pretty much any test that use the base class and spins up the application (given there is no trigger action in the test itself)?

Reply:
Correct, there's not much of a point to this test anymore.  This was one of the first tests I wrote, before I had written any specific RA tests, and it is now redundant.  I've removed it.